### ISSUE TITLE
issues about test on laneatt

### ISSUE THREAD

**Issue by Auroradsy** [voldemortX/pytorch-auto-drive#98]
I would like to test my tusimple dataset on laneatt, but I cannot find pretrained laneatt '.pt' model in the modelzoo of this repository(just have two for culane dataset based on resnet18 and resnet34 backbone), so I try to train a new one, but another file needed, 'tusimple_anchors_freq.pt'.

So could you please tell me how to do my test then? Or do you have pretrained laneatt model or suitable 'tusimple_anchors_freq.pt' that can be offered?

Thank you for your such powerful repository and thanks for your patience!

**Comment by voldemortX** [comment]
The re-implementation of LaneATT has not yet succeeded on TuSimple. 

anchors should be available from the LaneATT official repo.

**Comment by Auroradsy** [comment]
> 

Thank you, did you mean that the implementation for LanaATT in this repository is not completed, even if I got anchors from the official repo?

**Comment by voldemortX** [comment]
Yes they are untested. 
Maybe @cedricgsh can provide some further details about LaneATT.

**Comment by Auroradsy** [comment]
OK! I will have a try after I got anchors!
Thanks again for your patience!   :)

**Comment by cedricgsh** [comment]
The laneatt config has a laneatt_tusimple config file. You can trainging laneatt  on tusimple by that config file. But you have to put the `tusimple_anchors_freq.pt` in main director. The `tusimple_anchors_freq.pt`  can be found in `https://github.com/lucastabelini/LaneATT/tree/main/data`.
The re-implemented accuracy has almost aligned, but FP and FN are totally differently with official result. And I didn' t figure it 
out. @Auroradsy
